SofaKernel/framework/sofa/simulation/MechanicalMatrixVisitor.h

```diff
@@ -250,6 +241,29 @@ class SOFA_SIMULATION_CORE_API MechanicalAddSubMBK_ToMatrixVisitor : public Mech
 
     //Masses are now added in the addMBKToMatrix call for all ForceFields
 
+};
+
+/** Apply projective constaints of the whole scene */
+class SOFA_SIMULATION_CORE_API MechanicalAddProjectiveConstraint_ToMatrixVisitor : public MechanicalVisitor
```

Comment:
"apply" rather than "add" would be more comprehensible => MechanicalApplyProjectiveConstraint_ToMatrixVisitor